lib/redacted_struct.rb

```diff
@@ -57,4 +51,74 @@ def pretty_print(q)
     }
   end
   # rubocop:enable all
+
+  alias_method :to_s, :inspect
+
+  def base_name_for_inspection
+    if is_a? Struct
+      "struct"
+    elsif is_a? Data
```

Comment:
Since we only have 2 parent classes for now, and `Data` may not be defined, let's just make it a plain `else`?
```suggestion
    else
```